public/src/client/topic.js

```diff
@@ -197,10 +197,21 @@ define('forum/topic', [
 			}
 		}
 
-		var currentBookmark = localStorage.getItem('topic:' + ajaxify.variables.get('topic_id') + ':bookmark');
-
-		if (!currentBookmark || parseInt(postIndex, 10) >= parseInt(currentBookmark, 10)) {
-			localStorage.setItem('topic:' + ajaxify.variables.get('topic_id') + ':bookmark', postIndex);
+		var bookmarkKey = 'topic:' + ajaxify.variables.get('topic_id') + ':bookmark';
+		var currentBookmark = localStorage.getItem(bookmarkKey);
+//		console.log('localStorage.getItem: current bookmark is ' + currentBookmark);
+
+		if (!currentBookmark || parseInt(postIndex, 10) > parseInt(currentBookmark, 10)) {
```

Comment:
But we need to ` app.removeAlert('bookmark') ` in both cases, ` > ` and ` >= `
Otherwise that annoying alert won't go away in some cases. Especially it will be inconvenient when you're clicking on that link (right side) http://i.imgur.com/omoX0jF.png